client/src/Pages/Friends.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function Friends() {
+  return <Wrapper></Wrapper>;
```

Comment:
Is this page still under construction? I was wondering why it only consists of a Wrapper..

Reply:
Yes it is